portal/static/js/profile.js

```diff
@@ -228,17 +228,31 @@
                     return {"done": true};
                 });
             },
-            setTopLevelOrgs: function(data) {
-                if (data && data.careProvider) {
-                    var orgTool = this.getOrgTool();
+            setTopLevelOrgs: function(callback) {
+                callback = callback || {};
```

Comment:
Should that default be an anonymous function (`function()`) instead of an object (`{}`)?